app/services/partner_routing_service.rb

```diff
@@ -28,8 +29,15 @@ def route_from_source_param
     end
   end
 
-  def vita_partner_from_zip_code
-    # @routing_method = :zip_code
+  def route_from_zip_code
+    return false unless @zip_code.present?
+
+    vita_partner = VitaPartnerZipCode.where(zip_code: @zip_code).first&.vita_partner
```

Comment:
Right now, this will always assign to the _first_ vita partner based on ID if there are multiple VPs with a particular zip code. Seems like we'll need to make this more sophisticated over time, but it might be okay to start with. 

IF we're already dealing with overlapping zip codes, we def should handle this before merging. We could do so easily by doing the where, and then randomly selecting one of the VPs from the list and assigning to them. Then, once we've got some capacity numbers we can start to assign based on relative capacity, too.